app/lib/ignore.json

```diff
@@ -4,6 +4,6 @@
   "cf-tabs",
   "cf-tables",
   "cf-demo",
-  "cf-colors",
-  "cf-core"
+  "cf-core",
+  "cf-colors"
```

Comment:
I don't think the order matters (looks like there was no particular order before), but we could avoid this diff if you put it back exactly the way it was.